project.py

```diff
@@ -462,9 +585,10 @@ def start_processing(self, input_file):
 
 
 def main(input_file, slide_number=0, start_frame=0, check_interval=30,
-         sim=0.70):
+         sim=0.60, compare_func='dhash', dbg=True):
```

Comment:
Sorry, please ignore my previous comment, I have not read the function properly.